### ISSUE TITLE
Chat head not working

### ISSUE THREAD

**Issue by Anspri143** [Gurjot95/DirectChat-Feedback#4]
What the hell is going on I dont understand..  Man this is too much..  I am regretting now after buying your premium feature.. Everytime the Chat head dont work.. But after I switch off and then Switch on my mobile then only it works and again after few hours it stops working..  Really disgusting yaar...  Please fix it

**Comment by Gurjot95** [comment]
I am sorry for your issues. However it is not clear to me what's causing
the issue for you but I am releasing an update soon which should fix most
issues. If it still doesn't, let me know.

Regards
Gurjot Singh

**Comment by Anspri143** [comment]
No no please Bro I request you to Solve my problem..  I need it.. This was
a very helpful app to me but now its turning into disaster..  You have to
fix my problem personally..  Reply please if You want to help me out..  I
will explain my problem briefly

**Comment by Gurjot95** [comment]
Did the problem only started after you made in-app purchase?

**Comment by Anspri143** [comment]
No after buying the premium feature,  10-15 days it was working fine
superbly..  But now its not..  He actual problem is I have enabled the chat
head for whatsapp and The chat head works fine only if I once switch off
and then switch on my mobile..  Otherwise it is not working..   Also after
restarting my device and a few hours of good running of this app then again
the problem starts..  Chat head dont work..  The floating I mean that
bubble didn't come to my screen whenever I receive messages from
whatsapp..  So please man I request you to help me out..  I had also
enabled the background running of this application but still It dont
work..  The chat head I mean that bubble automatically get dissapeared
after a few hours

**Comment by Gurjot95** [comment]
I understand, Can you please share your order details when you made the purchase of premium feature, I will take a closer look into this.

**Comment by Anspri143** [comment]
I had purchased on July 1.. Please check it

**Comment by Anspri143** [comment]
Bro... Long time no reply..  Please reply man..  I need your help..  I had
sent what you have told..  So plzz help..

**Comment by Gurjot95** [comment]
Hey I am working on it. Please be patient and allow me some time to reply.
I work alone on this project along with other things so sometimes it can
take time to respond and please don't call 5 hours of no response as "long
time" as like I said, it's an individual project so things takes time.
Thanks for understanding.

Regards

**Comment by Anspri143** [comment]
😢😢😢.. Bro I didn't mean that..  But I actually love Your Direct chat app
and its a most important app in my Applist..  It makes my work sooo easy..
So Please Bro..  You can take Your own time to Make it bug free but Please
Keep me in you mind that I just love Your App and seriously I need it..

**Comment by Gurjot95** [comment]
Hopefully everything is working well now with latest update

**Comment by ShreyasSSN** [comment]
Exactly..It happens even after the Premium version...PRO version was perfect... i dont know what changes you have made....

**Comment by creativetrendsapps** [comment]
> What the hell is going on I dont understand.. Man this is too much.. I am regretting now after buying your premium feature.. Everytime the Chat head dont work.. But after I switch off and then Switch on my mobile then only it works and again after few hours it stops working.. Really disgusting yaar... Please fix it

Wow. You're being a jerk. The dev doesn't have to "fix your problem" immediately. Who the heck do you think you are? You paid for pro? Good. That doesn't mean you get to be a jerk because something may not be working 100%. Take your entitlement somewhere, and try to be more respectful.
